### DISCUSSION TITLE
Limit per task

### DISCUSSION THREAD

**Discussion by M-Ahall** [semaphoreui/semaphore#848] [Q&A] [answered]
Hi,

In an earlier version, I could add JSON for limiting a single run of a task template, like ["--limit", "foo.example"], but this seems to be gone. Is there another way to easily limit a single run?

Thanks!

**Comment by fiftin** [comment, marked as answer]
Sorry, I moved you question to Q&A incorrectly. It is UI issue. I will fix it soon, but now you should check following checkbox in Template edit form:

<img width=500 src="https://user-images.githubusercontent.com/914224/152433414-c2424df2-230f-411e-9d42-fa5d0ddf1c88.gif">

After that you can use Advanced setting in Task.

**Comment by macau23** [comment]
I am interested in this too. It would mean I could reduce the number of inventories from 20 to 1.

**Reply by ansibleguy** [reply]
See: [Feature - Limit as own Field](https://github.com/ansible-semaphore/semaphore/issues/1357)
